src/js/modules/wallet/controllers/sweep-coins-modal/sweep-coins-modal.ctrl.js

```diff
@@ -90,15 +97,26 @@
         $scope.startSweepingBIP44 = function() {
             $scope.working = true;
             $scope.discovering = true;
-            sweeperService.bip44Sweep(($scope.form.mnemonic || "").trim().replace(/  +/g, ' '), options).then(function (result) {
+            sweeperService.bip44Sweep(($scope.form.mnemonic || "").trim().replace(/  +/g, ' '), options).then(function (results) {
                 $scope.working = false;
                 $scope.discovering = false;
 
                 sweeperService.submitDebugInfo();
 
-                if (result) {
+                // Some are positive balance transactions, some below dust value
+                var somePositiveBalances = results.some(function (result) {
+                    return !!result;
+                });
+
+                // Add to absolute value to display before sending
+                results.map(function (result) {
+                    $scope.displaySweepData.totalValue += result.totalValue;
+                    $scope.displaySweepData.totalFeePaid += result.feePaid;
+                });
+
+                if (results && results.length && somePositiveBalances) {
```

Comment:
I notice this part, and the next part below are very similar.. could make the same function perhaps. 

sweeperService.wifSweep(WIFs, options).then($scope.displayResult)
sweeperService.bip44Sweep(seed, options).then($scope.displayResult)

Reply:
Done, `then` now calls `processSweepResults()`